lib/editor/util/validation.js

```diff
@@ -184,30 +192,28 @@ export function validate (
       const isNotLat = value > 90 || value < -90
       if (isNotLat) {
         reason = 'Field must be valid latitude.'
-        return {field: name, invalid: isOptionalAndEmpty || isNotLat, reason}
+        return {field: name, invalid: isNotLat, reason}
       }
-      if (entity && entity.location_type >= 2) {
+      if (!locationType || locationType < 2) {
```

Comment:
Check the GTFS reference for the location types, I think the condition should be `<= 2`: https://developers.google.com/transit/gtfs/reference#stopstxt